package.json

```diff
@@ -10,16 +10,19 @@
     "thunk",
     "test"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/es2015/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/es2015/index.d.ts",
   "files": [
     "dist/*"
   ],
   "scripts": {
     "clean": "rimraf dist",
     "lint": "eslint src/**/*.ts",
     "prebuild": "npm run clean",
-    "build": "tsc",
+    "build": "run-p build:es2015 build:esm",
+    "build:es2015": "tsc --module es2015 --target es2015 --outDir dist/es2015",
+    "build:esm": "tsc --module es6 --target es6 --outDir dist/esm",
```

Comment:
I'm confused. [ES6 and ES2015 are the same thing](https://bytearcher.com/articles/es6-vs-es2015-name/).